asserts/model.go

```diff
@@ -707,6 +738,20 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 			grade = ModelGrade(gradeStr)
 		}
 
+		storageSafetyStr, err := checkOptionalString(assert.headers, "storage-safety")
+		if err != nil {
+			return nil, err
+		}
+		if storageSafetyStr != "" && !strutil.ListContains(validStorageSafeties, storageSafetyStr) {
+			return nil, fmt.Errorf("storage-safety for model must be %s, not %q", strings.Join(validStorageSafeties, "|"), storageSafetyStr)
```

Comment:
```suggestion
			return nil, fmt.Errorf("storage-safety for model must be one of %s, not %q", strings.Join(validStorageSafeties, "|"), storageSafetyStr)
```

though the grade error is `"grade for model must be %s, not %q"`, so maybe not worth it unless we change the other one too.

Reply:
I prefer the compact variant,   with "one of" I think we would have to switch to comma as separator or something